shared/src/commonMain/kotlin/org/hyperskill/app/step_quiz/domain/analytic/ProblemOnboardingModalHiddenHyperskillAnalyticEvent.kt

```diff
@@ -0,0 +1,44 @@
+package org.hyperskill.app.step_quiz.domain.analytic
+
+import org.hyperskill.app.analytic.domain.model.hyperskill.HyperskillAnalyticAction
+import org.hyperskill.app.analytic.domain.model.hyperskill.HyperskillAnalyticEvent
+import org.hyperskill.app.analytic.domain.model.hyperskill.HyperskillAnalyticPart
+import org.hyperskill.app.analytic.domain.model.hyperskill.HyperskillAnalyticRoute
+import org.hyperskill.app.analytic.domain.model.hyperskill.HyperskillAnalyticTarget
+import org.hyperskill.app.step_quiz.presentation.StepQuizFeature
+import org.hyperskill.app.step_quiz_fill_blanks.model.FillBlanksMode
+
+/**
+ * Represents a hidden analytic event of the problem onboarding modal.
+ *
+ * JSON payload:
+ * ```
+ * {
+ *     "route": "/learn/step/1",
+ *     "action": "hidden",
+ *     "part": "parsons_problem_onboarding_modal",
```

Comment:
in such comments we usually add all possible values like `parsons_problem_onboarding_modal | fill_blanks_problem_onboarding_modal`